src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -1470,6 +1470,7 @@ public override SecurityToken ReadToken(string token)
         /// <param name="token">A 'JSON Web Token' (JWT) in JWS or JWE Compact Serialization Format.</param>
         /// <param name="validationParameters">A <see cref="TokenValidationParameters"/>  required for validation.</param>
         /// <returns>A <see cref="TokenValidationResult"/></returns>
+        [Obsolete("Synchronous wrapper over an asynchronous method")]
```

Comment:
add the aka.ms link https://aka.ms/IdentityModel/7-breaking-changes

maybe add a little more text to the obsolete message. 
I would add `[Obsolete("better message", false)]` so a warning appears in the IDE, but no build breaking.